frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -105,36 +116,71 @@ func replyJSON(w http.ResponseWriter, obj interface{}, statusCode int) error {
 	return encoder.Encode(obj)
 }
 
+func createCVD(c Config, om OperationManager, uadr UserArtifactsManager) *createCVDHandler {
```

Comment:
`newCreateCVDHandler` is probably better. `createCVD` implies the CVD is created in this function, which it isn't.